wallet/txrules/poolfees.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2016 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// Package txrules provides transaction rules that should be followed by
+// transaction authors for wide mempool acceptance and quick mining.
+package txrules
+
+import (
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/decred/dcrd/blockchain"
+	"github.com/decred/dcrd/chaincfg"
+	"github.com/decred/dcrutil"
+)
+
+const maxPoolFee = 10000.0
+
+// TestPoolFee tests to see if a pool fee is a valid percentage from 0.01% to
+// 100.00%.
+func TestPoolFee(fee float64) error {
```

Comment:
I'd avoid starting this with Test as it could cause confusion with a unit test.  I'd also say FeePercentage instead of Fee.